README.md

```diff
@@ -63,53 +63,50 @@ confluent consume test1 --value-format avro --max-messages 5 --property print.ke
-The following command builds the image using the `Dockerfile-confluenthub` specification and tags that image with `confluentinc/kafka-connect-datagen:0.1.5` (be sure to use the correct datagen connector version in the label).
+This project provides several Dockerfiles that you can use to create Docker images with this connector.  A few of these images are provided in [Dockerhub](https://hub.docker.com/r/cnfldemos/kafka-connect-datagen) already and you may not need to build the images locally.
 
+You can create a local build of the Docker image by running:
 ```bash
-docker build . -f Dockerfile-confluenthub -t confluentinc/kafka-connect-datagen:0.1.5
+make build-local
 ```
+This will build the connector from source and create a local image with an aggregate version number.  The aggregate version number is the kafka-connect-datagen connector version number and the Confluent Platform version number separated with a `-`.   The local kafka-connect-datagen version number is defined in the `pom.xml` file, and the Confluent Platform version defined in the `Makefile` (example `0.1.5-5.3.1`).
 
-### Build connector from latest code
+### Install connector from Confluent Hub
 
-Alternatively, you may build and install the `kafka-connect-datagen` connector from latest code.
-Here we use `v0.1.5` to reference the git tag for the `0.1.5` version, but the same pattern works for all released versions.
-Be sure to use the same version in the Docker image tag (e.g., `confluentinc/kafka-connect-datagen:0.1.5`) that you checked out (e.g., `v0.1.5`).
-
-```bash
-git checkout v0.1.5
-mvn clean package
-docker build . -f Dockerfile-local -t confluentinc/kafka-connect-datagen:0.1.5
-```
+You may install into your Docker image the `kafka-connect-datagen` connector from [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).  The `Dockerfile-confluenthub` file has an example of how this can be accomplished.  The `Makefile` has targets to build an image from the standard `cp-kafka-connect` and the `cp-server-connect-operator` base image if running Confluent Platform on Kubernetes.
 
 ### Run connector in Docker Compose
 
 Here is an example of how to run the `kafka-connect-datagen` with Docker Compose.
-If you used a different Docker image tag, be sure to use that here instead of `confluentinc/kafka-connect-datagen:0.1.5`.
+If you wish to use a different Docker image tag, be sure to modify appropriately in the `docker-compose.yml` file.
 
 ```bash
 docker-compose up -d --build
 curl -X POST -H "Content-Type: application/json" --data @config/connector_pageviews.config http://localhost:8083/connectors
 docker-compose exec connect kafka-console-consumer --topic pageviews --bootstrap-server kafka:29092  --property print.key=true --max-messages 5 --from-beginning
 ```
 
-### Publish Docker Image
+### Publish Docker Images
```

Comment:
How about we move "Publish" into its own section (not part of Install and Run).  Reasoning:

1. It's going to be a workflow for just for admins (us)
2. It reduces the muddiness of these sections.